.build/docker/_docker_run.sh

```diff
@@ -26,10 +26,14 @@
 #
 ################################
 
+[ $DEBUG ] && set -x
+
 # variables, with defaults
 [ "x${cassandra_dir}" != "x" ] || cassandra_dir="$(readlink -f $(dirname "$0")/../..)"
 [ "x${build_dir}" != "x" ] || build_dir="${cassandra_dir}/build"
+[ "x${m2_dir}" != "x" ] || m2_dir="${cassandra_dir}/build/m2"
```

Comment:
this is problematic.  

we want the `~/.m2` directory to be used, so to re-use the already downloaded .m2/repository 
(we don't want to download everything every time unnecessarily)

Reply:
I understand the rationale but the Home directory had some problems, like being unwritable on jenkins agent or something like that. Maybe stash m2 things? Is there any another possible approach?

Reply:
the problem is we want to re-use the `.m2/repository/` between runs, not just between stages… 

e.g. the post-commit CI on ci-cassandra.a.o

Reply:
We can use preserved stashes between runs, another option will be artifacts. I'll explore possible options.